generate-vendor-tarball.sh

```
@@ -1,17 +1,24 @@
+        "$@"
+    else
+        "$@" >/dev/null 2>&1
+    fi
+}
 
+# Helper function to find out if the required tools are present.
 check_required_tools() {
-    local tools=("cargo" "rpmspec" "spectool" "tar" "patch")
+    # Ensure ~/.cargo/bin is in PATH (cargo install places binaries there)
+    export PATH="${CARGO_HOME:-$HOME/.cargo}/bin:$PATH"
+    local tools=("make", "cargo" "rpmspec" "spectool" "tar" "patch")
```

Shellcheck (0.11.0)

[warning] Use spaces, not commas, to separate array elements.

(SC2054)